numpy/core/setup.py

```diff
@@ -392,7 +389,13 @@ def visibility_define(config):
 
 def configuration(parent_package='',top_path=None):
     from numpy.distutils.misc_util import Configuration, dot_join
-    from numpy.distutils.system_info import get_info
+    from numpy.distutils.system_info import (get_info, blas_opt_info,
+                                             lapack_opt_info)
+
+    # Accelerate is buggy, disallow it. See also numpy/linalg/setup.py
+    for opt_order in (blas_opt_info.blas_order, lapack_opt_info.lapack_order):
+        if 'accelerate' in opt_order:
+            opt_order.remove('accelerate')
```

Comment:
Is there a reason we don't simply remove `accelerate_info` ? From a backward compatibility perspective, I don't think it would make sense to continue using `accelerate_info` in any other package ?

Reply:
There used to be quite a few packages using `blas_opt` or `lapack_opt`. Accelerate is actually less buggy and easier to use that picking a random version of OpenBLAS, hence keeping `accelerate_info` for a while longer could make sense.

That said, a code search of the likes of Statsmodels and PyAMG shows that they've stopped using `*_opt`, so the impact of doing it now is likely to be very limited.

Reply:
I would be happy to remove it, but then we might need to do a 19.1 to revert the change. Perhaps it should be a separate PR to make reverting easier?